packages/editor-ui/src/components/SettingsLogStreaming/EventDestinationSettingsModal.ee.vue

```diff
@@ -180,20 +180,23 @@ import { useNDVStore } from '../../stores/ndv';
 import { useWorkflowsStore } from '../../stores/workflows';
 import ParameterInputList from '@/components/ParameterInputList.vue';
 import NodeCredentials from '@/components/NodeCredentials.vue';
-import { IMenuItem, INodeUi, ITab, IUpdateInformation } from '../../Interface';
+import type { IMenuItem, INodeUi, ITab, IUpdateInformation } from '../../Interface';
+import type {
+	IDataObject,
+	INodeCredentials,
+	NodeParameterValue,
+	MessageEventBusDestinationOptions,
+} from 'n8n-workflow';
```

Comment:
Should enums be also imported as type? If so, `MessageEventBusDestinationTypeNames` should be included